input/haleu/optimization/once-through/soga_tuning/soga_tuning_template.in

```diff
@@ -0,0 +1,49 @@
+#Optimization of Scenatio 7 for minimizing HALEU
+
+environment 
+  tabular_data
+    tabular_data_file = './tuning_{{counter}}.dat'
+  output_file = './tuning_{{counter}}.out'
+  write_restart = './tuning_{{counter}}.rst'
+
+method
+  soga
+    max_function_evaluations = 500
+    population_size = {{pop_size}}
+    mutation_type {{mutation_type}}
+    mutation_rate {{mutation_rate}}
+    crossover_type shuffle_random 
+    crossover_rate {{crossover_rate}}
+    convergence_type best_fitness_tracker
+    fitness_type merit_function
+      constraint_penalty {{constraint}}
+
+model
+  single
+
+variables
+  active all
+  discrete_design_range = 4 
+    lower_bounds 0 0 0 0 
+    upper_bounds 100 100 100 50
```

Comment:
Are these like presets? If so, did you need to specify all of them in `input/haleu/optimization/once-through/min_waste.in`?

Reply:
These are the upper and lower bounds of values to consider for each of these variables, and are user defined. The order corresponds to the variable order in one of the lines below this section. The `discrete_design_range` specification allows these variables to vary within these ranges, but must be integers.

Reply:
Oh, ok. I think I was overthinking this one. I thought it was a template that the above ones were inheriting, but that's not the case.